operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -893,17 +906,58 @@ public NextAction apply(Packet packet) {
 
   // pre-conditions: DomainPresenceInfo SPI
   // "principal"
-  private static Step bringAdminServerUp(Domain dom, Step next) {
-    return StorageHelper.insertStorageSteps(dom, Step.chain(bringAdminServerUpSteps(dom, next)));
+  private static Step bringAdminServerUp(DomainPresenceInfo info, Step next) {
+    return StorageHelper.insertStorageSteps(
+        info.getDomain(), Step.chain(bringAdminServerUpSteps(info, next)));
   }
 
-  private static Step[] bringAdminServerUpSteps(Domain dom, Step next) {
+  private static class ReadSituConfigMapStep extends ResponseStep<V1ConfigMap> {
+    private final DomainPresenceInfo info;
+
+    ReadSituConfigMapStep(DomainPresenceInfo info) {
+      this.info = info;
+    }
+
+    @Override
+    public NextAction onFailure(Packet packet, CallResponse<V1ConfigMap> callResponse) {
+      System.out.println(
+          "ReadSituConfigMapStep.onFailure callResponse.getStatusCode: "
+              + callResponse.getStatusCode());
+      return callResponse.getStatusCode() == CallBuilder.NOT_FOUND
+          ? onSuccess(packet, callResponse)
+          : super.onFailure(packet, callResponse);
+    }
+
+    @Override
+    public NextAction onSuccess(Packet packet, CallResponse<V1ConfigMap> callResponse) {
+      V1ConfigMap result = callResponse.getResult();
+      System.out.println("ReadSituConfigMapStep.onSuccess result: " + result);
```

Comment:
Should these System.out.println be replaced with LOGGER?

Reply:
Yes, LOGGER.finer

Reply:
Removed or replaced the System.out.println debug statements. Just pushed the changes